src/scripts/plugin/components/ads-label-loader.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+if (videojs.Component) {
```

Comment:
How'd you come up with this logic?  I'm assuming you saw it being used elsewhere?